Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:carbon
+
+WORKDIR /opt/facility-cache
+
+# Copy source
+COPY . ./
+
+# Install dependencies
+COPY package.json ./
```

Please could you also add the lock file here to keep dependencies identicle.

```suggestion
COPY package.json npm-shrinkwrap.json ./
```

---

@Zooloo2014 Minor, please add a blank line between the commands

---

@Zooloo2014 Is there a difference between `.` and `./` ? If not could you please change this to `.` to be consistent with the Validator and Orchestrator.

---

/ Is needed when the destination is a folder and there is more than one file in the source